Makefile

```diff
@@ -8,11 +8,11 @@
 NAME     = gophernicus
 PACKAGE  = $(NAME)
 BINARY   = $(NAME)
-VERSION  = 3.1
+VERSION  = 3.1.1
```

Comment:
Nope 3.1 isn't released yet, so we'll keep this as 3.1 and merge this change into 3.1.